internal/config/task.go

```diff
@@ -24,6 +25,16 @@ func buildTask(def *TaskDefinition, lc *loaderContext) (*task.Task, error) {
 	t.ResetContext = def.ResetContext
 
 	t.Env = variables.FromMap(def.Env).Merge(t.Env)
+	t.EnvFile = utils.NewEnvFile(func(e *utils.Envfile) {
+		if def.Envfile != nil {
+			e.Exclude = def.Envfile.Exclude
+			e.Include = def.Envfile.Include
+			e.Modify = def.Envfile.Modify
+			e.Path = def.Envfile.Path
+			e.Quote = def.Envfile.Quote
+			e.ReplaceChar = def.Envfile.ReplaceChar
+		}
```

Comment:
Is that really the right way to do this, is there no clone or similar?

Reply:
I suppose we already have mergo as a dependency - so we could use that...